lib/deaf_grandma.rb

```diff
@@ -19,9 +19,21 @@ def run!
 
 
   def speak(input)
-
-    #Implement your code here <<<<<<<<<
-
+    #Implement your code here <<<<<<<<
+
+    if !input.upcase!
```

Comment:
Try to avoid using destructive methods.  `unless input == input.upcase` preserves `input` for any later usage.